level2/main.tf

```diff
@@ -52,9 +45,9 @@ data "aws_ami" "amazonlinux" {
 }
 
 resource "aws_launch_configuration" "web_config" {
-  image_id             = data.aws_ami.amazonlinux.id
-  instance_type        = "t2.micro"
-  key_name             = "main"
+  image_id      = data.aws_ami.amazonlinux.id
+  instance_type = "t2.micro"
+  #key_name             = "main"
```

Comment:
Please don't leave commented-out lines. It's a bad style practice

Reply:
I forget to remove it. will do that now